# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/unit/xpcshell.ini
# Commit: 521c9e019678
# Full Hash: 521c9e019678f584590f63473416d40e03f33342
# Author: Valentin Gosu <valentin.gosu@gmail.com>
# Date: 2023-03-15 04:10:55
# Regressor Bug: 1805372
# File Overlap Count: 1
# Description:
#   Bug 1805372 - Pass effectiveTRRMode to nsHttpChannel from socketTransport/DnsAndConnectSocket r=necko-reviewers,kershaw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D164539
# ==============================================================================

diff -r dfb39ca05529 -r 521c9e019678 netwerk/test/unit/xpcshell.ini
--- a/netwerk/test/unit/xpcshell.ini	Tue Mar 14 15:29:49 2023 +0200
+++ b/netwerk/test/unit/xpcshell.ini	Tue Mar 14 12:59:40 2023 +0000
@@ -757,3 +757,4 @@
 [test_websocket_500k.js]
 skip-if = verify
 run-sequentially = node server exceptions dont replay well
+[test_trr_noPrefetch.js]
